### PR TITLE
Registration stats

### PR DESCRIPTION
**Pull request by OmeGak**
Depends on PR https://github.com/indico/indico/pull/2155

### CODE REVIEW

File: indico/modules/events/registration/models/registrations.py [hunk 1]
```diff
@@ -574,8 +585,15 @@ def _build_storage_path(self):
         return Config.getInstance().getAttachmentStorage(), path
 
     def render_price(self):
+<<<<<<< b95bc2f00a8a5fc18c0fb9a6408b4879955e8c78
```
Comment (mic4ael) [inline indico/modules/events/registration/models/registrations.py]: ups?

**Comment by ThiefMaster** [comment]
Broken for accommodation field with nothing selected

**Comment by OmeGak** [comment]
Fixed for accommodation fields with nothing selected

File: indico/modules/events/registration/controllers/management/regforms.py [hunk 2]
```diff
@@ -182,6 +184,17 @@ def _process(self):
                                                     regform=self.regform)
 
 
+class RHRegistrationFormStats(RHManageRegFormBase):
+    """Display registration form stats page"""
+
+    def _process(self):
+        regform_stats = [OverviewStats(self.regform)]
+        more_stats = [AccommodationStats(x) for x in self.regform.active_fields if x.input_type == 'accommodation']
+        regform_stats += more_stats
```
Comment (ThiefMaster) [inline indico/modules/events/registration/controllers/management/regforms.py]: what's the point of assigning it to `more_stats` first?

File: indico/modules/events/registration/templates/management/regform.html [hunk 3]
```diff
@@ -93,4 +93,9 @@
             </div>
         </div>
     </div>
+    <div class="toolbar right">
+        <a class="i-button icon-chart" href="{{ url_for('.regform_stats', regform) }}">
+            {% trans %}Results{% endtrans %}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/management/regform.html]: Wouldn't "Stats" be better? "Results" makes sense for a survey but in a registration it sounds strange

File: indico/modules/events/registration/templates/management/regform_stats.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+                    {% if head.type == 'str' -%}
+                        <th {%- if head.colspan > 1 %} colspan="{{ head.colspan }}"{% endif %}>{{ head.data }}</th>
+                    {%- endif -%}
+                {% endfor %}
+                </tr>
+            </thead>
+        {%- endif %}
+        {%- if table.rows %}
+            <tbody>
+            {%- for row_type, row in table.rows %}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/management/regform_stats.html]: should have one more level of indentation (probably it was already wrong in jacques' code though)

File: indico/web/templates/_statistics.html [hunk 5]
```diff
@@ -22,20 +22,22 @@
     </div>
```
Comment (ThiefMaster) [inline indico/web/templates/_statistics.html]: What about moving this template inside the registration module?

Comment (OmeGak) [inline indico/web/templates/_statistics.html]: It's not used only in the registration module.

**Pull request closed without merging**